src/Compiler/Webpack/ConfigurationBuilder.js

```diff
@@ -56,6 +59,14 @@ class ConfigurationBuilder {
         };
 
         const host = new NodeJsAsyncHost();
+
+        let webpackExtraConfigs = [];
+        for (const config of this._webpackConfigPaths) {
+            if (fs.existsSync(config)) {
```

Comment:
If the file does not exist an exception should be thrown

Reply:
👍